src/esw/drive/odrive_control.py

```diff
@@ -600,21 +580,54 @@ def _update(self) -> None:
             self._bridge_on_event(ODriveEvent.ODRIVE_ERROR_EVENT)
 
 
+class Application(object):
+    """Manages the ODriveBridge objects and keeps track of communication.
+    :param _bridges: A list of bridges that are contain data for the ODrives controlling
+        the front, middle, and back wheels."""
+
+    def __init__(self):
+
+        rospy.init_node(f"odrive_control")
+        self._bridges = [ODriveBridge("front"), ODriveBridge("middle"), ODriveBridge("back")]
+        rospy.Subscriber("drive_vel_cmd", DriveVelCmd, self._drive_vel_cmd_callback)
+
+    def run(self):
+        """Runs the publish data loop and watchdog loops for all bridges"""
+        for bridge in self._bridges:
+            threading._start_new_thread(bridge.ros_publish_data_loop, ())
```

Comment:
nit: typically better to use `Thread` objects

Reply:
Kk yea ill look into how that works